### ISSUE TITLE
[dropdown] input focus out when `minCharacters` more than one char in api mode

### ISSUE THREAD

**Issue by ziaratban** [fomantic/Fomantic-UI#1089]
hi
focus out of input when `minCharacters` more than one char in api mode.

**step by step**
inject below js code in home page of [Fomantic-UI](https://fomantic-ui.com/)
```js
var myModal = $(
'<div class="ui modal">\
  <i class="close icon">\</i>\
  <div class="header">\
    Modal Title\
  </div>\
  <div class="content">\
    <div class="ui search selection dropdown">\
      <input type="hidden" name="gender">\
      <i class="dropdown icon">\</i>\
      <div class="default text">\Gender</div>\
      <div class="menu">\
      </div>\
    </div>\
  </div>\
  <div class="actions">\
    <div class="ui button">\Cancel</div>\
    <div class="ui button">\OK</div>\
  </div>\
</div>'
);

myModal.modal({
    onVisible : function(){
        $('.ui.dropdown',this).dropdown({
            minCharacters : 3,
            apiSettings: {
              // this url parses query server side and returns filtered results
              url: '//api.semantic-ui.com/tags/{query}',
            },
        });
      
    }
}).modal('show');
```
after code injected , modal opens.
then search `abc` in dropdown and you see `No results found.`.
now press `CTRL + A` (select all) and type only `a` character  , **finally focus out occur**

## Version
2.7.8

**Comment by lubber-de** [comment]
Confirmed,I made a jsfiddle testcase. It does not even have to be a modal and it also happens, when you just try to press backspace.
http://jsfiddle.net/s4p0henv/

![backspace_vanish](https://user-images.githubusercontent.com/18379884/66671623-c2271b80-ec5c-11e9-90ec-6101c986f7ca.gif)

**Comment by lubber-de** [comment]
Fixed by #1095
